### ISSUE TITLE
udp config error

### ISSUE THREAD

**Issue by sun-iot** [rossmcdonald/influxdb#22]
```yaml
###
###[[udp]]
###
### Controls the listener for InfluxDB line protocol data via UDP.
###

  # TODO: Including the udp table prevents service from starting (even if disabled)
  # [[udp]]
  # enabled = {{ influxdb_udp_enabled }}
  # bind-address = "{{ influxdb_udp_bind_address }}"
  # database = "{{ influxdb_udp_database }}"
  # batch-size = {{ influxdb_udp_batch_size }}
  # batch-timeout = "{{ influxdb_udp_batch_timeout }}"
```

**Comment by sun-iot** [comment]
The correct configuration should be this, the influxdb.conf.j2 you provided has a problem with the UDP configuration

**Comment by rossmcdonald** [comment]
Fixed with #23
